rootfs/led-daemon/led-daemon

```diff
@@ -14,9 +14,19 @@ GRAIN=100000        # Our timing granularity: 100ms in microseconds
 # LED initialisation
 
 # The LED GPIO numbers are determined from the physical GPIO numbers (50 for the
-# STA status LED, 51 for the DIA diagnostic LED), but an offset of 906 is added
+# STA status LED, 51 for the DIA diagnostic LED), but an offset is added
 # for somewhat mysterious reasons.
-GPIO_BASE=906
+ZYNQ_GPIO_BASE=906
+ZYNQMP_GPIO_BASE=338
+if [ -e /sys/class/gpio/gpiochip${ZYNQ_GPIO_BASE} ]; then
+    GPIO_BASE=${ZYNQ_GPIO_BASE}
+elif [ -e /sys/class/gpio/gpiochip${ZYNQMP_GPIO_BASE} ]; then
+    GPIO_BASE=${ZYNQMP_GPIO_BASE}
+else
+    echo "Unknown GPIO chip"
+    exit -1
+fi
+
```

Comment:
Let's pick the LED configuration up from the device tree or a config file.

If we do this right, then the `led-daemon` fix can be merged into master before the rest of this merge, because we can add the definitions in rootfs for now.  Think I quite like the idea of doing this.